packages/ui/src/config/mantine.ts

```diff
@@ -1,6 +1,33 @@
-import { MantineTheme } from "@mantine/core";
+import { MantineThemeOverride } from "@mantine/core";
 
-export const mantineConfig: Partial<MantineTheme> = {
-  colorScheme: "dark"
+import { Tuple, DefaultMantineColor } from "@mantine/core";
```

Comment:
this can be one import

```ts
import { type Tuple, type DefaultMantineColor, type MantineThemeOverride } from "@mantine/core"
// OR
import type { Tuple, DefaultMantineColor, MantineThemeOverride } from "@mantine/core"
```

Reply:
noted